docker/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '2'
+
+services:
+  db:
+    image: mysql:5.6
+    container_name: ${Site_Name}-db
+    command: "mysqld --character-set-server=utf8 --collation-server=utf8_general_ci --max_allowed_packet=1G"
+    environment:
+      MYSQL_DATABASE: "${MySQL_DB}"
+      MYSQL_USER: "${MySQL_User}"
+      MYSQL_PASSWORD: "${MySQL_Password}"
+      MYSQL_ROOT_PASSWORD: "${MySQL_Root_Password}"
+    volumes:
+      - ${MySQL_Data_Dir}:/var/lib/mysql
+    ports:
+      - "${DB_Port}:3306"
```

Comment:
@cioan - this is what I was referring to in that other PR.  This is the correct way to do it.